src/main/java/org/jitsi_modified/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -37,6 +38,12 @@
      */
     private final RTPEncodingDesc[] rtpEncodings;
 
+    /**
+     * Allow the lookup of an encoding by the SSRC of a received packet.  Note
+     * that multiple SSRCs in this map may point to the same encoding.
+     */
```

Comment:
the key is no longer purely the ssrc, right? let's update the comment to reflect that/describe what the key is now `(ssrc | tid << 32)`

Reply:
btw, is this now vp8 specific?  i also wondered if using `VideoRtpPacket.qualityIndex` make more sense than `ssrc | tid << 32`? then it could be codec-agnostic, but i think this is actually how we get the qualityIndex...

Reply:
ok, i see now we use `ssrc | tid << 32` for vp8, but just the ssrc otherwise.  i think it'd still be good to put a comment somewhere describing why we do this for vp8